### ISSUE TITLE
Host stable and dev validators

### ISSUE THREAD

**Issue by effigies** [bids-standard/bids-validator#86]
This is a redux of https://github.com/bids-standard/legacy-validator/issues/1227, in light of #85, also with branching diagrams.

I propose that we have the following URLs:

* Stable: https://bids-standard.github.io/bids-validator/
* Development: https://bids-standard.github.io/bids-validator/dev/ (I'm open to alternatives to `dev`)

The goal here would be for the stable release to reliably correspond to the latest version of the BIDS specification, while `dev` can be updated and used continuously, with a warning that it contains experimental features that may be removed before a stable release.

We would use patch releases to fix bugs in the stable validator, but not to add any new features, which will be largely helped by #85 allowing us to upgrade the schema without adopting unreleased features.

```mermaid
gitGraph TB:
  branch web order: 2
  branch web-dev order: 3
  checkout main
  commit id: "v1.15.0.devN"
  branch stable order: 1
  commit id: "v1.15.0" tag: "v1.15.0"
  checkout web
  commit
  branch gh-pages order: 4
  checkout main
  merge stable id: "v1.16.0.dev1"
  checkout web-dev
  commit
  checkout gh-pages
  merge web-dev
  checkout main
  commit id: "v1.16.0.dev2"
  checkout web-dev
  commit
  checkout gh-pages
  merge web-dev
  checkout stable
  commit id: "v1.15.1" tag: "v1.15.1"
  checkout web
  commit
  checkout gh-pages
  merge web
  checkout main
  merge stable id: "v1.16.0.dev3"
  checkout web-dev
  commit
  checkout gh-pages
  merge web-dev
```

**Comment by effigies** [comment]
This is done, except that we are using `main` and `dev` instead of `stable` and `main`. We also are not pushing to gh-pages on a continuous basis, since I found a way (#96) to avoid committing the contents by building from two branches.

These are currently triggered by release or manual trigger. Is that good enough, or do we want to automate more frequent builds, and if so what should be the `stable` target?

**Comment by effigies** [comment]
Done in #96, patched in 6e301bf867b86193287acac4f305339a66ec3420.
